index.d.ts

```diff
@@ -0,0 +1,3 @@
+export default logger;
+
+declare function logger(): (state: any, emitter: any) => void;
```

Comment:
What do you think about an argument `opts` ? :)

Reply:
That's my mistake! Updated to add object interface for `opts`.